### ISSUE TITLE
Skript mit SEAT/CUPRA nutzbar

### ISSUE THREAD

**Issue by xerion3800** [tillsteinbach/WeConnect-cli#15]
Ist das Skript auch mit SEAT bzw. CUPRA nutzbar und wenn ja was muss ich anpassen?

**Comment by tillsteinbach** [comment]
Ich fürchte nicht. Die URLs werden anders sein. Ich habe leider keinen SEAT, daher ist das für mich extrem schwer umzusetzen.

**Comment by xerion3800** [comment]
Wenn ich dein Projekt richtig verfolgt habe dann nutzt du als "Basis" die iobroker Umsetzung. Da geht auf jeden Fall Seat. Ich versuche mal einen Vergleich zu machen wenn du mir die relevanten Stellen nennen könntest.

**Comment by tillsteinbach** [comment]
Ne, der ioBroker teil ist raus. Das hatte zu viele limits.
Ich benutze direkt die neue API von VW. Die Frage ist ob es die bei Seat auch schon gibt. Hat Seat schon ein MEB Fahrzeug am Markt? Und dann ist noch die Frage ob es auch die "alten" Fahrzeuge supported.

**Comment by xerion3800** [comment]
SEAT hat momentan nur PHEV´s im Portfolio. Bei CUPRA sieht es schon anders aus dort kommt der Born. Ich habe zusätzlich zu einem Kona EV noch einen Cupra PHEV. Zu den "alten" könnte man den SEAT Mii zählen. Wir denn im ioBroker Adapter noch eine "veraltete" Version genutzt?

**Comment by tillsteinbach** [comment]
Beim ioBroker Adapter wählt man das interface aus. Das neue interface ist "id", für den skoda enyaq gibt es jetzt "skodae" wenn ich es richtig gesehen habe. Für den Cupra habe ich noch nichts gesehen.

**Comment by xerion3800** [comment]
Bei Cupra nimmt man einfach Seat, da Cupra eine Seat Tochter ist.

**Comment by tillsteinbach** [comment]
Jo, das ist dann aber das „alte“ Interface. Man muss mal schauen was geht wenn es eine App für den Born gibt.

**Comment by tillsteinbach** [comment]
„Steuere deinen CUPRA Born über die My CUPRA App – bald verfügbar!“ das wäre dann schon mal die richtige App.

**Comment by tillsteinbach** [comment]
Ich verschiebe das mal in das WeConnect-python project, weil es da umgesetzt werden müsste und wir warten mal ab dass die App kommt. Dann müsste nur noch das Problem gelöst werden dass ich temporär Zugriff auf ein Fahrzeug haben müsste...
